### PR TITLE
add jupyter_ascending

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raybellwaves** [comment]
nothing provides requested edlib >=1.3.8,<2.0.0

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=432400&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=926

**Comment by raybellwaves** [comment]
https://github.com/conda-forge/staged-recipes/pull/17351

File: recipes/jupyter_ascending/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - poetry
+  run:
+    - attrs >=19.0.0
+    - edlib >=1.3.8,<2.0.0
+    - ipywidgets >=7.5.1,<8.0.0
+    - jsonrpcclient >=3.3.5,<4.0.0
+    - jsonrpcserver >=4.1.2,<5.0.0
+    - jupytext >=1.3.4,<2.0.0
+    - loguru >=0.4.1
+    - poetry
```
Comment (dopplershift) [inline recipes/jupyter_ascending/meta.yaml:31]: Is `poetry` really needed at run time?
```suggestion
```

Comment (raybellwaves) [reply recipes/jupyter_ascending/meta.yaml]: can't remember. let's find out.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**